pkg/util/error_list.go

```diff
@@ -27,10 +27,10 @@ import (
 
 // This helper implements the error interface for []error, but prevents
 // accidental conversion of []error to error.
-type errorList []error
+type ErrorList []error
```

Comment:
Without a type to cast []error to, there is no way to get the slice back.  So anyone who wants to use a slice of errors to return multiple has to declare their own type.  I'm not sure having lots of type foo []error scattered all over the code is worth it (if there's another way to make the []error cast legal I'd be happy to change to it, or if we want to create Errors() []error).


> On Jan 4, 2015, at 1:28 AM, Tim Hockin <notifications@github.com> wrote:
> 
> In pkg/util/error_list.go:
> 
> > @@ -27,10 +27,10 @@ import (
> >  
> >  // This helper implements the error interface for []error, but prevents
> >  // accidental conversion of []error to error.
> > -type errorList []error
> > +type ErrorList []error
> NAK.
> 
> The whole point of the recent transformations here was to simplify - callers just use []error and SliceToError() (which is still poorly named).
> 
> —
> Reply to this email directly or view it on GitHub.
>